shiny/render/_data_frame.py

```diff
@@ -275,6 +285,18 @@ def data_view(self, *, selected: bool = False) -> pd.DataFrame:
         else:
             return self._data_view_all()
 
+    data_view_info: reactive.Calc_[DataViewInfo]
```

Comment:
Add public-facing examples that use this.

For example: an app where there is an interactive table, and below it, a code output that prints out:
- The values of `sort` and `filter`
- A data frame using `rows`, which reflects exactly the sorted and filtered data that the user is seeing in the interactive table, and also has an extra column indicating whether that row is selected.
- Also add a second code output that just prints the value of `.data_view(selected=TRUE)`.